src/analysis/retail/mage/frost/talents/RayOfFrost.tsx

```diff
@@ -1,13 +1,52 @@
 import { SHATTER_DEBUFFS } from 'analysis/retail/mage/shared';
 import { formatPercentage } from 'common/format';
 import SPELLS from 'common/SPELLS';
-import TALENTS from 'common/TALENTS/mage';
-import { SpellLink } from 'interface';
+import { TALENTS_MAGE } from 'common/TALENTS/mage';
+import { SpellIcon, SpellLink } from 'interface';
 import Analyzer, { Options } from 'parser/core/Analyzer';
 import { SELECTED_PLAYER } from 'parser/core/EventFilter';
 import Events, { CastEvent, DamageEvent, GetRelatedEvents } from 'parser/core/Events';
-import { When, ThresholdStyle } from 'parser/core/ParseResults';
+import { ThresholdStyle, When } from 'parser/core/ParseResults';
 import Enemies from 'parser/shared/modules/Enemies';
+import { RoundedPanel } from 'interface/guide/components/GuideDivs';
+import { BoxRowEntry, PerformanceBoxRow } from 'interface/guide/components/PerformanceBoxRow';
+import { explanationAndDataSubsection } from 'interface/guide/components/ExplanationRow';
+import { GUIDE_CORE_EXPLANATION_PERCENT } from 'analysis/retail/mage/frost/Guide';
+import CastEfficiencyBar from 'parser/ui/CastEfficiencyBar';
+import { GapHighlight } from 'parser/ui/CooldownBar';
+import { QualitativePerformance } from 'parser/ui/QualitativePerformance';
+import Spell from 'common/SPELLS/Spell';
+import { Fragment } from 'react';
+import { ChevronIcon } from 'interface/icons';
+import styled from '@emotion/styled';
+
+const SequenceContainer = styled.div`
+  display: inline-flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  height: 24px;
+
+  & svg {
+    transform: rotate(-90deg);
+    height: 18px;
+    margin-top: calc(24px / 2 - 18px / 2);
+  }
+
+  & img.icon {
+    height: 24px;
+  }
+`;
+
+const SpellSeq = ({ spells }: { spells: Spell[] }) => (
+  <SequenceContainer>
+    {spells.map((spell, index, array) => (
+      <Fragment key={index}>
+        <SpellIcon spell={spell} key={index} />
+        {index < array.length - 1 && <ChevronIcon />}
+      </Fragment>
+    ))}
+  </SequenceContainer>
+);
```

Comment:
This is a useful UI snippet, please move it to a new standalone file in `src/parser/ui/...` so it can be used by everyone with copy pasting.

Reply:
Done, extracted from Mistweaver Monk too (where I tooked from).